test/integration/error_output.cc

```diff
@@ -219,3 +222,103 @@ TEST(ErrorOutput, ElementErrorOutput)
   // Check nothing has been printed
   EXPECT_TRUE(buffer.str().empty()) << buffer.str();
 }
+
+////////////////////////////////////////
+// Test Model class for sdf::Errors outputs
+TEST(ErrorOutput, ModelErrorOutput)
```

Comment:
Any reason for removing the `PrintConfigErrorOutput` test?

Reply:
It was not removed, I think this PR was made before [PrintConfig One](https://github.com/gazebosim/sdformat/pull/1098), although the later went in first.

Reply:
It's there now after merging the changes from `main`.